README.md

```diff
@@ -59,52 +59,52 @@ harder to contribute to.
 
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-* [Installation](#installation)
```

Comment:
I think your editor may have messed this stuff up. If you could run `npx kcd-scripts format` before you commit that will ensure that everything's formatted properly. Thanks!

Reply:
It most assuredly did 😬